src/MainBanner.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { TdsBanner, TdsLink } from '@scania/tegel-react';
+
+interface MainBannerProps {
+  style?: React.CSSProperties;
+  className?: string;
+  onClose?: () => void;
+}
+
+const MainBanner: React.FC<MainBannerProps> = ({ style, className, onClose }) => {
```

Comment:
Just realised, that we usually been doing it like this with interfaces
```suggestion
const MainBanner = ({ style, className, onClose }: MainBannerProps) => {
```

Reply:
Ah yes, I would say using React.FC is a more complete way of doing things though.